ios/PurchasesHybridCommon/PurchasesHybridCommonSwiftTests/PurchasesHybridCommonSwiftTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PurchasesHybridCommonSwiftTests.swift
+//  PurchasesHybridCommonSwiftTests
+//
+//  Created by Andrés Boedo on 4/13/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import XCTest
```

Comment:
I didn't add new tests here yet, we might just stick to the ones in PurchasesHybridCommon